counterpoll/main.py

```diff
@@ -241,6 +242,45 @@ def disable():
     configdb.mod_entry("FLEX_COUNTER_TABLE", "PG_WATERMARK", fc_info)
     configdb.mod_entry("FLEX_COUNTER_TABLE", BUFFER_POOL_WATERMARK, fc_info)
 
+# ACL counter commands
+@cli.group()
+@click.pass_context
+def acl(ctx):
+    """  ACL counter commands """
+    ctx.obj = ConfigDBConnector()
+    ctx.obj.connect()
+
+@acl.command()
+@click.argument('poll_interval', type=click.IntRange(1000, 30000))
+@click.pass_context
+def interval(ctx, poll_interval):
+    """
+    Set ACL counters query interval
+    interval is between 1s and 30s.
+    """
+
+    fc_group_cfg = {}
+    fc_group_cfg['POLL_INTERVAL'] = poll_interval
+    ctx.obj.mod_entry("FLEX_COUNTER_TABLE", ACL, fc_group_cfg)
+
+@acl.command()
+@click.pass_context
+def enable(ctx):
+    """ Enable ACL counter query """
+
+    fc_group_cfg = {}
+    fc_group_cfg['FLEX_COUNTER_STATUS'] = ENABLE
+    ctx.obj.mod_entry("FLEX_COUNTER_TABLE", ACL, fc_group_cfg)
```

Comment:
@stepanblyschak FLEX_COUNTER_TABLE = "FLEX_COUNTER_TABLE" ?

Reply:
This means refactoring of existing code. I am often asked to seperate such change in a different pr